packages/queries/src/query.ts

```diff
@@ -3,6 +3,8 @@ import { Where } from './types/Where';
 import { Select } from './types/Select';
 import { Column } from './types/Column';
 import { Table } from './types/Table';
+import { DbSchema } from './types/schemas';
```

Comment:
This is not following conventions @jimezesinachi.

Notice that all other types in /queries are single-type-per-file and the file is named identically as the exposed type.